controllers/bpfman-operator/configmap.go

```diff
@@ -73,7 +72,7 @@ func (r *BpfmanConfigReconciler) SetupWithManager(mgr ctrl.Manager) error {
 }
 
 func (r *BpfmanConfigReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	r.Logger = log.FromContext(ctx)
+	r.Logger = ctrl.Log.WithName("configMap")
```

Comment:
all reconciller object has Logger
```golang
type ReconcilerCommon struct {
	client.Client
	Scheme *runtime.Scheme
	Logger logr.Logger
}
```
so we just need to initialize in the operator and agent main and use it here and all places below ?

Reply:
Would could, but this way there is context.